doc/source/io.rst

```diff
@@ -134,6 +134,14 @@ usecols : array-like, default ``None``
   inferred from the document header row(s). For example, a valid `usecols`
   parameter would be [0, 1, 2] or ['foo', 'bar', 'baz']. Using this parameter
   results in much faster parsing time and lower memory usage.
+as_recarray : boolean, default ``False``
+  DEPRECATED: this argument will be removed in a future version. Please call
+  ``pd.read_csv(...).to_records()`` instead.
+
+  Return a NumPy recarray instead of a DataFrame after parsing the data. If
```

Comment:
Of course.  The test I added <a href="https://github.com/pydata/pandas/pull/13373/files#diff-fe58f4e9e077f2d5c7811982556792bfR1395">here</a> speaks for itself.